app/controllers/users_controller.rb

```diff
@@ -0,0 +1,27 @@
+class UsersController < ApplicationController
+
+  def index
+   render json: @users = User.all
```

Comment:
Are you running the project and testing it? This will not render anything, just a JSON. You will have to do something like this:
```
    @users = User.all

    respond_to do |format|
      format.html
      format.json { render json: @users }
    end
```